fatiando/mesher/mesh.py

```diff
@@ -388,6 +388,117 @@ def copy(self):
         return cp.deepcopy(self)
 
 
+class PointMesh(object):
+    """
+    A generic mesh of 3D point sources (spheres of unit volume).
+
+    Use this as a 1D list of :class:`~fatiando.mesher.Sphere`.
+
+    Grid points are ordered like a C matrix, first each row in a column, then
+    change columns. In this case, the x direction (North-South) are the rows
+    and y (East-West) are the columns.
```

Comment:
This paragraph doesn't really make sense anymore for this mesh type.

Reply:
Removed